chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -55,4 +55,48 @@ ovs-vsctl set-fail-mode <%= bridge %> standalone
 echo "Reloading wicked configuration for all interfaces"
 wicked ifreload all
 
+<% if @compute_node %>
+
+echo "Upgrading services needed to act as nova-compute node"
+
+zypper --non-interactive up openstack-nova-compute
+# This is ugly, but fixing neutron spec to upgrade to correct version of pecan is even uglier:
+zypper --non-interactive up python-pecan
```

Comment:
Actually, with recent changes done by @toabctl it should be fixed. I've yet to test it...